src/main/resources/db/migration/V1_26__rename_consumer_acquirer.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE consumer_subscription
+    RENAME COLUMN consumer_id TO acquirer_id;
+
+ALTER TABLE consumer_subscription_enrichment_field
+    RENAME COLUMN consumer_subscription_id TO acquirer_subscription_id;
+
+ALTER TABLE event_data
+    RENAME COLUMN consumer_subscription_id TO acquirer_subscription_id;
+
+ALTER TABLE consumer
+    RENAME CONSTRAINT event_consumer_pkey TO acquirer_pkey;
+ALTER TABLE consumer
+    RENAME CONSTRAINT event_consumer_consumer_name_key TO acquirer_name_key;
```

Comment:
It would be conventional to use `acquirer_acquirer_name_key` since constraint names are Schema scoped

Reply:
Its acquirer.name so I think this is fine:)